src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_Geometry.cs

```diff
@@ -132,5 +138,51 @@ public void EmptyGeometry_CheckBounds()
 		{
 			Console.WriteLine(Geometry.Empty.Bounds);
 		}
+
+		[TestMethod]
+		public async Task When_EllipseGeometry()
+		{
+#if __MACOS__
+			Assert.Inconclusive(); // MACOS interpret colors differently
```

Comment:
curious, how so ?

Reply:
It was not the right message because that test was not implemented in MacOS in the Sample app to begin with.